docs/labs/handling-errors.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://best.openssf.org/assets/css/style.css">
+<link rel="stylesheet" href="checker.css">
+<script src="js-yaml.min.js"></script>
+<script src="checker.js"></script>
+<link rel="license" href="https://creativecommons.org/licenses/by/4.0/">
+
+<!-- See create_labs.md for how to create your own lab! -->
+
+<!-- Sample expected answer -->
+<script id="expected0" type="plain/text">
+throw new Error("Division by zero is not allowed");
+</script>
+<!--
+-->
+<script id="expected1" type="plain/text">
+  return a / b;
+</script>
+
+<script id="expected2" type="plain/text">
+  try {
+    const result = divide(10, 2);
+    console.log("Result:", result);
+  } catch (error) {
+    console.error("Error:", error.message);
+}
+   
+
+</script>
+
+<!-- Full pattern of correct answer -->
+<script id="correct0" type="plain/text">
+\s* throw new Error \( ("(.*?)" | '(.*?)' | `(.*?)`) \) ; \s*
+</script>
+<script id="correct1" type="plain/text">
+\s* return .\s* a \s* \/ \s* b \s* ; \s*
+</script>
+<script id="correct2" type="plain/text">
+  \s* try \s* { \s* 
+    const result  \s* =  \s* divide \( \s* 10  \s* ,  \s* 2  \s* \)  \s* ;
+    console.log \(  \s* ("Result:" | 'Result:' | `Result:`)  \s* ,  \s* result  \s* \)  \s* ;
+    \s* } \s* catch \(  \s* error  \s* \)  \s* {  \s*
+      \s* console.error \(  \s* ("Error:" | 'Error:' | `Error:`)  \s* ,  \s* error.message  \s* \)  \s* ;  \s*
+      \s* \} \s*
+  </script>
+
+<script id="info" type="application/yaml">
+---
+hints:
+- index: 0
+  present: "(Throw|THROW|New|NEW|error|ERROR)"
+  text: JavaScript is case-sensitive. use throw new Error(...).
+  examples:
+  - - "  Throw new Error(\"Division by zero is not allowed\");"
+  - - "  THROW new Error(\"Division by zero is not allowed\");"
+  - - "  throw New Error(\"Division by zero is not allowed\");"
+  - - "  throw NEW Error(\"Division by zero is not allowed\");"
+  - - "  throw new error(\"Division by zero is not allowed\");"
+  - - "  throw new ERROR(\"Division by zero is not allowed\");"
+- index: 0
+  absent: "throw"
+  text: Try using the throw keyword to raise an exception.
```

Comment:
I suggest being more specific since some learners won't necessarily have a solid knowledge of JavaScript syntax. E.g., append text something like this:

> E.g., throw new Error(...)